objcgen/generator.cs

```diff
@@ -40,14 +40,9 @@ public virtual void Generate ()
 
 		protected virtual void Generate (ProcessedAssembly a)
 		{
-			foreach (var t in a.Assembly.GetTypes ()) {
```

Comment:
I believe this is dead code (_unlike_ the code above it). Instead of updating I just deleted it.